### PR TITLE
[UX SUGGESTION] Active section should be expanded on docs page load

### PR DESCRIPTION
**Pull request by corwintines**
Add method to open active section on left navbar in docs

Fixes: https://github.com/ethereum/geth-website/issues/164

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
🎉

File: src/utils/checkNavLinks.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { NavLink } from "../types";
+
+interface Props {
+  to?: string
+  items?: NavLink[]
+  pathCheck: string
+  tracker?: boolean
+}
+export const checkNavLinks = ({ to, items, pathCheck, tracker = false }: Props): boolean => {
```
Comment (nhsz) [inline src/utils/checkNavLinks.ts:9]: Why is `tracker` a param here? Not being used on function call. IMO makes more sense to define it as a let var inside the function (also a boolean param can't be reassigned later like in the conditionals below I think).

**Review by nhsz — COMMENTED** [review]
Looks good, left a comment about `tracker` param

File: src/utils/checkNavLinks.ts (hunk 1 above)
Comment (wackerow) [reply src/utils/checkNavLinks.ts]: Good catch, thanks @nhsz... patched!

**Review by nhsz — APPROVED** [review]

**Pull request merged**